cuda/private/repositories.bzl

```diff
@@ -232,4 +311,5 @@ def rules_cuda_dependencies(toolkit_path = None):
         ],
     )
 
-    local_cuda(name = "local_cuda", toolkit_path = toolkit_path)
+    local_cuda(name = "local_cuda", cuda_version = cuda_version, toolkit_path = toolkit_path)
```

Comment:
There's similar calls to local_cuda that need updating in the bzlmod module extension in cuda/extensions.bzl. You'll need to expose this same cuda_version option in the modules tag_class.